fdbclient/NativeAPI.actor.cpp

```diff
@@ -908,6 +980,12 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 			validateOptionValue(value, false);
 			networkOptions.slowTaskProfilingEnabled = true;
 			break;
+		case FDBNetworkOptions::SEND_DETAILED_HEALTH_METRICS:
+			validateOptionValue(value, true);
+			int sendDetailedHealthMetrics;
+			std::istringstream(value.get().toString()) >> sendDetailedHealthMetrics;
```

Comment:
if a user passes a non-number value, this will throw a non-fdb exception. Convert the error to invalid_option_value